doc/source/whatsnew/v0.24.0.txt

```diff
@@ -17,6 +17,8 @@ New features
 
 - ``ExcelWriter`` now accepts ``mode`` as a keyword argument, enabling append to existing workbooks when using the ``openpyxl`` engine (:issue:`3441`)
 
+- ``DataFrame.to_parquet()`` now accepts ``index`` as a keyword argument, allowing the user to optionally omit the dataframe's indexes from the resulting Parquet file. (:issue:`20768`)
+
```

Comment:
We generally do either the `whatsnew` entry OR the mini-section but not both.

cc @jreback

Reply:
Done.